hpc0.qmd

```diff
@@ -1,297 +1,1300 @@
+name: Introduction to Linux for HPC
 ---
 
-{{< include _title.qmd >}}
-{{< include _team.qmd >}}
 
-## Contact Details
+{{< include _donut.qmd >}}
 
-\
-ARC Website: {{< var rc.website >}}
 
-\
-General queries: {{< var rc.servicedesk >}}
+{{< include _title.qmd >}}
 
 
-## Training
+{{< include _team.qmd >}}
 
+
+## Training Themes
 ![](assets/img/hpc0/training.png)
 
-## ARC/HPC Training Portfolio
+::: {.notes}
+- Read slide:
+  - "We are developing and providing training in the following themes..."
+:::
 
-{{< var rc.training >}}
 
+## Training Portfolio {visibility="hidden"}
+- [Research Computing Website Training]({{< var rc.training >}})
 ![](assets/img/hpc0/portfolio.png)
 
-## Purpose of today
+::: {.notes}
+- Skipping/hiding slide until we refactor what training we are doing and produce a new outline...
+:::
+
+
+## Aims {.smaller}
+- Get aquainted.
+- Gain experience and build confidence in using a Linux command line.
+- Prepare for [HPC1: Introduction to High Performance Computing](https://arc.leeds.ac.uk/training/courses/hpc1/) (HPC1).
+- Get feedback from you about the training.
+
+::: {.notes}
+- Read slide:
+  - "Our training aims today are to..."
+- "Please introduce yourself on the hackpad...":
+  - If using Teams:
+    - Share hackpad link in the chat.
+    - Explain about using the chat.
+    - Maybe conduct a poll:
+      - Is everyone happy so far?
+  - If not using Teams and in a classroom, then hopefully everyone can use the hackpad link from the course reminder email that should have been sent recieved recently...
+- "Please interrupt if you are having difficulty hearing or following the presentation..."
+  - "Please ask questions, or ask for things to be clarified or repeated..."
+:::
+
+
+## Syllabus {.smaller}
+<!--
+This should match what is in:
+https://github.com/ARCLeeds/arcleeds.github.io/tree/master/_data/training_courses.yml
+https://github.com/ARCTraining/rc-lesson-plans/blob/main/hpc0/hpc0-lesson-plan.md
+-->
+- Logging in
+- Entering commands
+- Finding out about commands
+- File system navigation
+- Commands to list, create, copy, move and delete files
+- Hidden files
+- Command history
+- Control key combinations
+- Editing files
+- File permissions
+- Commands to explore and filter data
+- Wildcards
+- Shell scripting basics
+
+::: {.notes}
+- "This slide shows the syllabus of what we plan to go through."
+- "There is quite a lot to cover in 3 hours!"
+- "We'll aim to have 5 or 10 minute breaks roughly every 30 minutes."
+:::
 
-- Introduce the command line for new users
-- Hands on experience and confidence building
-- Basic interactions with files and directories: command line execution, directory and file basics, files properties.
-- File manipulation: inspecting and editing files, searching and filtering,
-- Scripting and workflows: scripting techniques for building workflows.
 
-# Introductions and Motivations
+## Logging in 1/3
+- [ARC Documentation >> Login]({{< var rc.login >}})
```

Comment:
Don't like the >>.  Could just be wordier, and say, ARC Documentation on logging on ?

Reply:
OK